File: src/main/java/com/github/brankale/jcolorspace/colorspaces/ColorSpaces.java
package com.github.brankale.jcolorspace.colorspaces;

import com.github.brankale.jcolorspace.colorspace.ColorSpace;
import com.github.brankale.jcolorspace.colorspace.rgb.Rgb;

public class ColorSpaces {
    private ColorSpaces() {
        // hide constructor
    }

    public static final ColorSpace CIE_LAB = new CieLab();
    public static final Rgb SRGB = new Srgb();
    public static final Rgb DCI_P3 = new DciP3();

}

File: src/main/kotlin/com/github/brankale/jcolorspace/colorspaces/ColorSpaces.kt
package com.github.brankale.jcolorspace.colorspaces

import com.github.brankale.jcolorspace.colorspace.ColorSpace
import com.github.brankale.jcolorspace.colorspace.rgb.Rgb

object ColorSpaces {
    @JvmField val CIE_LAB: ColorSpace = CieLab()
    @JvmField val SRGB: Rgb = Srgb()
    @JvmField val DCI_P3: Rgb = DciP3()
}
